### PR TITLE
feat: Added cart management page

### PR DESCRIPTION
**Pull request by Vishruta-Patil**
Completed cart management page

### CODE REVIEW

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    <title>magniZent Store</title>
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style/header.css" />
+    <link rel="stylesheet" href="./style/colors.css" />
+    <link rel="stylesheet" href="./style/reset.css" />
+    <link rel="stylesheet" href="./style/utility.css" />
+    <link rel="stylesheet" href="./style/cart.css" />
```
Comment (shivanigangadharan) [inline cart.html:16]: I think you can import all these into 1 components.css and then just import that file here.

**Review by anshikaa12 — APPROVED** [review]
clean code!
Looks good to merge

**Comment by Roshannair24** [comment]
Looks good to merge

File: style/utility.css [hunk 2]
```diff
@@ -56,6 +52,12 @@ button {
   font-weight: bold;
 }
 
+.outline-btn:hover,
```
Comment (shivanigangadharan) [inline style/utility.css:55]: You should also try to use components from your component library, like buttons..

**Review by Roshannair24 — APPROVED** [review]

**Review by shivanigangadharan — COMMENTED** [review]
A couple of suggestions, other than that the code looks good and the UI is also nice. 👍

**Pull request merged**